ruby/ql/test/query-tests/security/cwe-598/app/controllers/users_controller.rb

```diff
@@ -0,0 +1,22 @@
+class UsersController < ApplicationController
+
+  def login_get
+    password = params[:password] # BAD: route handler uses GET query parameters to receive sensitive data
```

Comment:
It think you best change the test case in one that uses `params[:password]` but with a non-sensitive variable name, and another with a non-sensitive query parameter name and a sensitive variable name.